tensorflow/lite/kernels/elementwise_test.cc

```diff
@@ -123,6 +134,15 @@ TEST(ElementWise, Square) {
   EXPECT_THAT(m.GetTensorShape(m.output()), ElementsAreArray({1, 1, 4, 1}));
 }
 
+TEST(ElementWise, IsFinite) {
+  ElementWiseOpIsFiniteModel m(BuiltinOperator_ISFINITE, {1, 1, 4, 1});
+  m.PopulateTensor<float>(m.input(), {1, std::nan("2"), std::nan("5"), -3.0});
```

Comment:
nit: since it's test against finite, better include infinity data as well.

Reply:
@renjie-liu , i have updated as per your suggestion.

Regards
Amit